Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -77,10 +81,34 @@ public class BellerophonManager: NSObject {
             }
         }
     }
-
+    
+    /// Updates the display with the given status.
+    ///
+    /// - Parameter status: Kill switch status to check if the kill switch view should be displayed.
+    @objc public func updateDisplay(_ status: BellerophonObservable) {
+        if status.forceUpdate() {
+            displayForceUpdate()
+        } else if status.apiInactive() {
+            displayKillSwitch()
+        } else {
+            dismissKillSwitchIfNeeded()
+        }
+    }
+    
+    /// Dismisses the kill switch window.
+    @objc public func dismissKillSwitchIfNeeded() {
```

Comment:
same comment for tests here since this is public now

Reply:
It was already being tested as an internal function